### ISSUE TITLE
Can't see posts from external instances, global feed is empty

### ISSUE THREAD

**Issue by mackiexx** [pixelfed/pixelfed#3679]
I can't see post from the global feed. When i search for the user manually, nothing shows on the user's page and followers ,following and post shows "0" even though on the main website, the user has many post and followers. I have the activitypub on

**Comment by holajsh** [comment]
Probably same issue as #3115.
This is imho a major showstopper for the adoption of the product.

**Comment by TiiFuchs** [comment]
I have the same problem. Can't see any posts of external instances. I can search and follow them, and they get a notification, but if they follow back, I get nothing...

There are no errors in the logs or the js console and I can't figure out if there is a configuration I missed, that need fixing...

**Comment by mitexleo** [comment]
I have the same problem  too

**Comment by ghost** [comment]
> I have the same problem. Can't see any posts of external instances. I can search and follow them, and they get a notification, but if they follow back, I get nothing...
> 
> There are no errors in the logs or the js console and I can't figure out if there is a configuration I missed, that need fixing...

Same effect here - searching them is okay, following them, too - but no updates are shown in my timeline, yet my instance just has my own pics in its stream only.

**Comment by vanlueckn** [comment]
I have the same problem

**Comment by mattlqx** [comment]
As well it doesn't scape their post/follow counts on the remote server...

<img width="294" alt="Screenshot 2022-12-17 at 10 48 23 AM" src="https://user-images.githubusercontent.com/739851/208254743-5e564470-2249-4e4d-bc0f-2bbc8ffc2425.png">

You're just left with confirmation that someone exists, and that's it.

**Comment by eobet** [comment]
I’m surprised this is still a problem. Is Pixelfed going the Bluesky route? Claiming to be distributed but then not fully supporting it?

I just followed my first user, and was confused to see that when I pasted their url into the search bar on pixelfed.social it appeared as if they had 0 posts, which I know when viewed from their pixelfed server isn’t true.

How is this not a massive showstopper which should be prioritized before adding any new features? Are we only meant to browse new photos, not older ones?
